pysat/instruments/icon_mighti.py

```diff
@@ -274,7 +274,7 @@ def clean(inst, clean_level=None):
     elif inst.tag[0:2] == 'te':
         # neutral temperatures
         mvar = 'Temperature'
-        if (clean_level == 'good') or (clean_level == 'dusty'):
+        if inst.clean_level in ['good', 'dusty']:
```

Comment:
Should 'good' be 'clean'?

Reply:
Yes. Sorry about my bad code in the area.